examples/etl/multiple_input_example.py

```diff
@@ -2,60 +2,70 @@
 from pyspark.sql import DataFrame
 from pyspark.sql.types import StructType, StructField, StringType
 
-from atc.etl import Extractor, Loader, MultiInputTransformer, Orchestration
+from atc.etl import Extractor, Loader, Orchestrator, Transformer
 from atc.spark import Spark
 
 
 class AmericanGuitarExtractor(Extractor):
     def read(self) -> DataFrame:
         return Spark.get().createDataFrame(
-            Spark.get().sparkContext.parallelize([
-                ('1', 'Fender', 'Telecaster', '1950'),
-                ('2', 'Gibson', 'Les Paul', '1959')
-            ]),
-            StructType([
-                StructField('id', StringType()),
-                StructField('brand', StringType()),
-                StructField('model', StringType()),
-                StructField('year', StringType()),
-            ]))
+            Spark.get().sparkContext.parallelize(
+                [
+                    ("1", "Fender", "Telecaster", "1950"),
+                    ("2", "Gibson", "Les Paul", "1959"),
+                ]
+            ),
+            StructType(
+                [
+                    StructField("id", StringType()),
+                    StructField("brand", StringType()),
+                    StructField("model", StringType()),
+                    StructField("year", StringType()),
+                ]
+            ),
+        )
 
 
 class JapaneseGuitarExtractor(Extractor):
+    def __init__(self):
+        super().__init__(dataset_key="japanese")
+
     def read(self) -> DataFrame:
         return Spark.get().createDataFrame(
-            Spark.get().sparkContext.parallelize([
-                ('3', 'Ibanez', 'RG', '1987'),
-                ('4', 'Takamine', 'Pro Series', '1959')
-            ]),
-            StructType([
-                StructField('id', StringType()),
-                StructField('brand', StringType()),
-                StructField('model', StringType()),
-                StructField('year', StringType()),
-            ]))
-
-
-class CountryOfOriginTransformer(MultiInputTransformer):
+            Spark.get().sparkContext.parallelize(
+                [("3", "Ibanez", "RG", "1987"), ("4", "Takamine", "Pro Series", "1959")]
+            ),
+            StructType(
+                [
+                    StructField("id", StringType()),
+                    StructField("brand", StringType()),
+                    StructField("model", StringType()),
+                    StructField("year", StringType()),
+                ]
+            ),
+        )
+
+
+class CountryOfOriginTransformer(Transformer):
     def process_many(self, dataset: {}) -> DataFrame:
-        usa_df = dataset['AmericanGuitarExtractor'].withColumn('country', f.lit('USA'))
-        jap_df = dataset['JapaneseGuitarExtractor'].withColumn('country', f.lit('Japan'))
+        usa_df = dataset["AmericanGuitarExtractor"].withColumn("country", f.lit("USA"))
+        jap_df = dataset["japanese"].withColumn("country", f.lit("Japan"))
```

Comment:
Just for extra niceness. Consider renaming the dataset keys to be more similar.

Reply:
e.g. "AmericanGuitars" and "JapaneseGuitars"

Reply:
I deliberately made them distinctly different to highlight the fact that the key was set manually to a new value.